### PR TITLE
feat: checkbox component

### PR DESCRIPTION
**Pull request by DamiBMS**
Description : 

- create MSCheckpoint component with styles files & storybook

Captura de pantalla 2023-06-12 a la(s) 16 06 30
Captura de pantalla 2023-06-12 a la(s) 16 06 17

### CODE REVIEW

File: src/components/checkbox/ms-checkbox.styles.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    label {
+        color: #37363B;
+        font-size: 17px;
+        line-height: 24px;
+    }
+
+    input[type=checkbox] {
+        margin-right: 8px;
+        accent-color: #6237D7;
```
Comment (Reynaldoqsbol) [inline src/components/checkbox/ms-checkbox.styles.ts:27]: Add TODOS where tokens are needed eg: `acent-color`

File: src/components/checkbox/ms-checkbox.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    description = '';
+
+    @property({ type: Boolean })
+    checked = false;
+
+    @property({ type: Boolean })
+    disabled = false;
+
+    render() {
+        return html`
```
Comment (Reynaldoqsbol) [inline src/components/checkbox/ms-checkbox.ts:23]:
```suggestion
        return  this.renderWithStyles(html`
```
It should be wrapped with this method to override customizations

**Review by Reynaldoqsbol — COMMENTED** [review]
0- add an image or figma ref 
1- use a single "checkbox" without typography as atom
2- add a `index.ts` in checobox folder to export everything in `ms-checkobx` eg `export * from './checkbox `
3 if linter is not working, replace in `.vscode/settings.json` with 
```{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "source.fixAll": true,
    "source.organizeImports": true
  }
}
```

File: src/components/checkbox/ms-checkbox.styles.ts (hunk 1 above)
Comment (DamiBMS) [reply src/components/checkbox/ms-checkbox.styles.ts]: Done!

File: src/components/checkbox/ms-checkbox.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { LitElement, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import { msCheckboxStyles } from "./ms-checkbox.styles";
+
+@customElement('ms-checkbox')
+export class MSCheckbox extends LitElement {
```
[the remark this answers is not in the document]
Comment (DamiBMS) [reply src/components/checkbox/ms-checkbox.ts]: Done!

File: src/components/checkbox/ms-checkbox.ts (hunk 2 above)
Comment (DamiBMS) [reply src/components/checkbox/ms-checkbox.ts]: Done!

**Review by Reynaldoqsbol — APPROVED** [review]

File: src/components/molecules/input-checkbox/ms-input-checkbox.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msInputCheckboxStyles } from './ms-input-checkbox.styles'
+
+@customElement('ms-input-checkbox')
+export class MSInputCheckbox extends WithTheme(LitElement) {
+  static styles = [msInputCheckboxStyles]
```
Comment (Reynaldoqsbol) [inline src/components/molecules/input-checkbox/ms-input-checkbox.ts:10]:
```suggestion
  static styles = msInputCheckboxStyles
```
could you check if it works in this way?

Comment (DamiBMS) [reply src/components/molecules/input-checkbox/ms-input-checkbox.ts:10]: Yeah! its works! Done!

**Pull request merged**